authors/settings/base.py

```diff
@@ -136,3 +139,46 @@
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'authors.apps.authentication.backends.JWTAuthentication',),
 }
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.environ['SOCIAL_AUTH_GOOGLE_OAUTH2_KEY']
```

Comment:
We have been using config instead of os.environ, would you change how we access environment variables for the sake of uniformity?